samples/command/Cargo.toml

```diff
@@ -21,6 +21,7 @@ serde = { workspace = true }
 serde_derive = { workspace = true }
 serde_json = { workspace = true }
 tokio = { workspace = true, features = ["macros", "rt-multi-thread"] }
+tokio-stream = { workspace = true }
```

Comment:
I need to separate the gRPC interface for all operations into several different groups: get/set/invoke, streaming, pub/sub, so that we don't need this in each sample.  This refactoring will be done at a later time.